README.md

```diff
@@ -17,26 +17,40 @@ API documentation is available at [https://hexdocs.pm/phoenix](https://hexdocs.p
 
 We appreciate any contribution to Phoenix. Check our [CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md) and [CONTRIBUTING.md](CONTRIBUTING.md) guides for more information. We usually keep a list of features and bugs [in the issue tracker][4].
 
-### Generating a Phoenix project from unreleased versions
+### Generating a Phoenix project from master
```

Comment:
this approach will work for unreleased versions as well as master, so I prefer the original heading. For example, you could checkout an unreleased branch and cd into installer and the same rules apply